hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestBulkDeleteCommand.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.hadoop.fs.shell;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.LinkedList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestBulkDeleteCommand {
+    private static Configuration conf;
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        conf = new Configuration();
+    }
+
+    @Test
+    public void testDefaults() throws IOException {
+        LinkedList<String> options = new LinkedList<>();
+        BulkDeleteCommand bulkDeleteCommand = new BulkDeleteCommand();
+        bulkDeleteCommand.processOptions(options);
+        assertTrue(bulkDeleteCommand.childArgs.isEmpty());
```

Comment:
prefer assertJ with description. otherwise, add a text message in the assert to explain the failure